src/spikeinterface/preprocessing/filter.py

```diff
@@ -123,14 +135,25 @@ def __init__(
             margin_ms=margin_ms,
             add_reflect_padding=add_reflect_padding,
             dtype=dtype.str,
+            direction=direction,
         )
 
 
 class FilterRecordingSegment(BasePreprocessorSegment):
-    def __init__(self, parent_recording_segment, coeff, filter_mode, margin, dtype, add_reflect_padding=False):
+    def __init__(
+        self,
+        parent_recording_segment,
+        coeff,
+        filter_mode,
+        margin,
+        dtype,
+        add_reflect_padding=False,
+        direction="forward-backward",
```

Comment:
I *think* it makes sense to remove the default here (also, from `add_reflect_padding` which was already there). I think this will always be called from the parent class where the default value is already set in the parent recording class constructer. @alejoe91 @samuelgarcia will know better though if for some reason defaults should be set on segment in some cases.

Reply:
for discussion in https://github.com/SpikeInterface/spikeinterface/issues/3243